Homeworks/homework_1st/hw1.py

```diff
@@ -0,0 +1,43 @@
+"""
+Given an array of size n, find the most common and the least common elements.
+The most common element is the element that appears more than n // 2 times.
+The least common element is the element that appears fewer than other.
+
+You may assume that the array is non-empty and the most common element
+always exist in the array.
+
+Example 1:
+
+Input: [3,2,3]
+Output: 3, 2
+
+Example 2:
+
+Input: [2,2,1,1,1,2,2]
+Output: 2, 1
+
+"""
+from typing import List, Tuple
+
+
+def major_and_minor_elem(list):
+    # create a dictionary to get only unique values
+    dictionary = dict.fromkeys(list, 0)
+
+    # set number of occurrences in the list
+    for value, _ in dictionary.items():
+        count = list.count(value)
+        dictionary.update({value: count})
+
+    # find the least common number
+    least_common = min(dictionary, key=dictionary.get)
+
+    most_common = list[0]
```

Comment:
I believe you can just do 
```py
most_common = max(dictionary, key=dictionary.get)
```

Reply:
oh right, assuming that the most common element always exist. thanks :)